src/components/footer/navigation-menu/navigation-menu.tsx

```diff
@@ -9,9 +12,25 @@ export const NavigationMenu = ({
   title,
   menuElements,
 }: NavigationMenuProps) => {
+  const [arrow, setArrow] = useState<string>("down");
+
+  const ArrowChange = (id) => {
+    console.log(id);
```

Comment:
If pull request is not yet ready to be checked then mark it as 'draft'

Reply:
Oki